nginx.conf

```diff
@@ -0,0 +1,74 @@
+worker_processes 1;
+
+events {
+    worker_connections 1024;
+}
+
+http {
+    sendfile on;
+    tcp_nopush on;
+    tcp_nodelay on;
+    keepalive_timeout 65;
+    types_hash_max_size 2048;
+    include /etc/nginx/mime.types;
+    default_type application/octet-stream;
+    
+    # Add some basic logging
+    log_format main '$remote_addr - $remote_user [$time_local] "$request" '
+                    '$status $body_bytes_sent "$http_referer" '
+                    '"$http_user_agent" "$http_x_forwarded_for"';
+    access_log /var/log/nginx/access.log main;
+
+    server {
+        listen 80;
+        server_name localhost;
+                
+        # Serve the static health check page
+        location = /check.html {
+            root /usr/share/nginx/html;
+            try_files /health.html =404;
+        }
+
+        # Route all API requests to the server
+        location /api/ {
+            # this will create problem in starting cmf-server on any port
```

Comment:
Is this statement correct ? Users now do not have a reason to change the server port as it is internal